BTSPocket/Controllers/Profile/ProfileViewController.swift

```diff
@@ -15,25 +15,40 @@ private enum ProfileSections: Int {
 
 class ProfileViewController: UIViewController {
     
-    // MARK:- Outlets
+    // MARK:- Outlets & Variables
     @IBOutlet weak private var tableView: UITableView!
     @IBOutlet weak var labelFullName: UILabel!
     @IBOutlet weak var imageProfile: UIImageView!
     @IBOutlet weak var labelPosition: UILabel!
     @IBOutlet weak var labelField: UILabel!
     
+    private var profileVM: ProfileViewModel?
+    
     override func viewDidLoad() {
         super.viewDidLoad()
+        self.profileVM = ProfileViewModel()
+        self.setUpProfile()
+        self.updateProfileData() 
+    }
+    
+    // MARK:- functions
+    func updateProfileData() {
+        self.profileVM?.getProfile({ [weak self] error in
+            if let error = error {
```

Comment:
unwrapped error is unused.
What about if error is due to server maintenance? should we logout user even when token is valid?

Reply:
I understand, if the error is 401, the section must be eliminated. But what should the app do with a 500 for example or a 408?

Reply:
@cristian-bts Generic error like 
"We are experimenting server issues. Please try again"